src/Test/Perf/util/test_util.csx

```diff
@@ -200,6 +216,41 @@ ProcessResult ShellOut(
     };
 }
 
+ProcessResult ShellOutUsingShellExecute(
```

Comment:
The code I have is here:  https://github.com/KevinH-MS/roslyn/blob/master/src/Test/Perf/infra/fetch_build.csx

(sorry, I realized after I typed that I hadn't merged it yet)

That said, I would still suggest that we don't robocopy during the test runner (it should be some sort of test setup script, like bootstrap.bat or fetch_build.csx).  Once I have the machine set up, I'd like to be able to run the tests without hitting the network for anything (running tests seems orthogonal from fetching tools).